packages/alfa-style/test/property/mask-position.spec.tsx

```diff
@@ -0,0 +1,227 @@
+import { test } from "@siteimprove/alfa-test";
+import { h } from "@siteimprove/alfa-dom";
+
+import { computed } from "../common.js";
+
+test("initial value is 0% 0%", (t) => {
+  const element = <div></div>;
+
+  t.deepEqual(computed(element, "mask-position"), {
+    value: {
+      type: "list",
+      separator: ", ",
+      values: [
+        {
+          type: "position",
+          horizontal: {
+            type: "side",
+            side: {
+              type: "keyword",
+              value: "left",
+            },
+            offset: {
+              type: "percentage",
+              value: 0,
+            },
+          },
+          vertical: {
+            type: "side",
+            side: {
+              type: "keyword",
+              value: "top",
+            },
+            offset: {
+              type: "percentage",
+              value: 0,
+            },
+          },
+        },
+      ],
+    },
+    source: null,
+  });
+});
+
+// TODO: The spec requires the computed value to be two lengths or percentages, not a keyword value.
+// E.g. the keyword `left` should be computes to `0% 50%` in Chrome and Firefox.
```

Comment:
**Note:** This may actually be a used value 🤔 [CSS spec](https://drafts.fxtf.org/css-masking/#the-mask-position) states that the computed value should be keywords + offsets. Anyway, it's probably OK as is if we don't need more 😄 (this may be something that ultimately needs to be fixed in the `Position` type)

The discrepancy note should be added to the source file, not just to the test.